test/voice-broadcast/components/VoiceBroadcastBody-test.tsx

```diff
@@ -111,22 +109,36 @@ describe("VoiceBroadcastBody", () => {
 
     describe("when displaying a voice broadcast recording", () => {
         beforeEach(() => {
-            mocked(shouldDisplayAsVoiceBroadcastRecordingTile).mockReturnValue(true);
+            renderVoiceBroadcast();
         });
 
         it("should render a voice broadcast recording body", () => {
-            renderVoiceBroadcast();
             screen.getByTestId("voice-broadcast-recording-body");
         });
+
+        describe("and the recordings ends", () => {
+            beforeEach(() => {
+                const stoppedEvent = mkVoiceBroadcastInfoEvent(VoiceBroadcastInfoState.Stopped);
+                const relationsHelper = mocked(RelationsHelper).mock.instances[5];
+                act(() => {
+                    // @ts-ignore
+                    mocked(relationsHelper.on).mock.calls[0][1](stoppedEvent);
```

Comment:
Don't ask :see_no_evil:  If this is really an issue I will put it on my clean up list.